src/arch/cuda/hybridAPI/gpumanager.h

```diff
@@ -100,6 +148,19 @@ struct GPUManager {
     // Number of PEs mapped to each device
     pes_per_device = -1;
 
+    // Device communication buffer
+    comm_buffer_size = 1 << 26; // 64MB by default
```

Comment:
Is there a way for users to override this?

Reply:
Yes there is the `+gpucommbuffer` command line argument